src/core/service/notifications/registry/handler.go

```diff
@@ -158,13 +161,16 @@ func (n *NotificationHandler) Post() {
 			}()
 
 			if autoScanEnabled(pro) {
-				last, err := clairdao.GetLastUpdate()
-				if err != nil {
-					log.Errorf("Failed to get last update from Clair DB, error: %v, the auto scan will be skipped.", err)
-				} else if last == 0 {
-					log.Infof("The Vulnerability data is not ready in Clair DB, the auto scan will be skipped, error %v", err)
-				} else if err := coreutils.TriggerImageScan(repository, tag); err != nil {
-					log.Warningf("Failed to scan image, repository: %s, tag: %s, error: %v", repository, tag, err)
+				artifact := &v1.Artifact{
```

Comment:
Will this Artifact be moved out of `/scan/***` package?

Reply:
I marked it as a TODO item, I think there will be a global `Artifact` model after OCI is ready. So far, we do not need to do the movement.

Reply:
I'll write up an issue to make sure it's done.

Reply:
Sure!